### DISCUSSION TITLE
제페토의 서버 시간을 스크립트로 가져오는 방법있나요?

### DISCUSSION THREAD

**Discussion by zlmrzlo** [naverz/zepeto-studio-kor#937] [5. Q&A]
혹시 제페토 서버시간을 가지고 올 수 있는 방법은 따로 없나요?

현실 세계의 시간으로 특정 시간마다 뭔가를 하고 싶어요.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 무빙 블록 멀티플레이 예제에 룸에서 흐른 시간에 따라 동기화가 되는 스크립트가 구현되어 있습니다.
https://github.com/naverz/world-movingblock-sample/blob/main/Assets/World.multiplay/index.ts
https://github.com/naverz/world-movingblock-sample/blob/main/Assets/Scripts/Multiplay/BlockMultiplay.ts
https://github.com/naverz/world-movingblock-sample/blob/main/Assets/Scripts/Multiplay/MultiMovingBlock.ts
참고 부탁 드립니다.

**Reply by zlmrzlo** [reply]
혹시 제페토 스크립트로 현실세계의 시간을 가지고 올 수도 있나요?

**Reply by Zepeto-TechEvangelist** [reply]
new Date() 를 사용해서 timestamp 형식의 시간을 받아올 수 있습니다. 
샘플 예제의 index.ts 스크립트 내에서 플레이어가 룸에 입장시 new Date()를 이용한 타임스탬프를 기록하는 부분을 참고 부탁 드립니다.

<img width="689" alt="스크린샷 2022-06-30 오후 1 42 14" src="https://user-images.githubusercontent.com/100118650/176594381-a2ff8965-162a-45ac-9a7c-3bdeae1b0923.png">

**Reply by zlmrzlo** [reply]
감사합니다 ㅠㅠ
